backend/cmd/headlamp.go

```diff
@@ -103,7 +105,7 @@ func StartHeadlampServer(config *HeadlampConfig) {
 
 		contextsFound, err := GetContextsFromKubeConfigFile(kubeConfigPath)
 		if err != nil {
-			log.Println("Failed to get contexts from", kubeConfigPath, err)
+			configError = fmt.Sprintf("Failed to get contexts from %s", kubeConfigPath)
```

Comment:
Maybe we could send the error as a `{ message: string, code: number }` so we can check for the code in the UI and show a nicer error message.

Reply:
@joaquimrocha i think the error string in itself is easy to manage because if it's not there than no error if it's there than there is an error why do we need the code???

Reply:
Because the code can tell us what the error is: e.g. if we use for example error codes, we can send the error that the server had when reading the config file, e.g. `ENOENT`. And thus we can quickly check if the error code is this in the frontend and make a better message for the user: `The kube config file was not found. Please read this documentation on how to set up K8s. [ Link to the docs ]`

If we only have the message, then we're configuring something that is to be facing the user (error message) from the backend, and we don't have to e.g. add JSX code to the error message in the backend.